scripts/run_gcosmos.sh

```diff
@@ -5,13 +5,77 @@ G_HTTP_ADDR=${G_HTTP_ADDR:-"26657"}
+./gcosmos config set app grpc.address localhost:0 --home ./test/val4
+
+./gcosmos config set app rest.enable false --home ./test/val2
+./gcosmos config set app rest.enable false --home ./test/val3
+./gcosmos config set app rest.enable false --home ./test/val4
+
+./gcosmos keys add val --no-backup --keyring-backend=test --home ./test/val1
+./gcosmos keys add val --no-backup --keyring-backend=test --home ./test/val2
+./gcosmos keys add val --no-backup --keyring-backend=test --home ./test/val3
+./gcosmos keys add val --no-backup --keyring-backend=test --home ./test/val4
+
+./gcosmos genesis add-genesis-account val 10000000stake --home ./test/val1 2>&1 > /dev/null 
+./gcosmos genesis add-genesis-account val 10000000stake --home ./test/val2 2>&1 > /dev/null
+./gcosmos genesis add-genesis-account val 10000000stake --home ./test/val3 2>&1 > /dev/null
+./gcosmos genesis add-genesis-account val 10000000stake --home ./test/val4 2>&1 > /dev/null
+
+VAL1ADDR=$(./gcosmos keys show val -a --keyring-backend=test --home ./test/val1)
+VAL2ADDR=$(./gcosmos keys show val -a --keyring-backend=test --home ./test/val2)
+VAL3ADDR=$(./gcosmos keys show val -a --keyring-backend=test --home ./test/val3)
+VAL4ADDR=$(./gcosmos keys show val -a --keyring-backend=test --home ./test/val4)
+
+./gcosmos genesis add-genesis-account ${VAL1ADDR} 10000000stake --home ./test/genesis 2>&1 > /dev/null
+./gcosmos genesis add-genesis-account ${VAL2ADDR} 10000000stake --home ./test/genesis 2>&1 > /dev/null
+./gcosmos genesis add-genesis-account ${VAL3ADDR} 10000000stake --home ./test/genesis 2>&1 > /dev/null
+./gcosmos genesis add-genesis-account ${VAL4ADDR} 10000000stake --home ./test/genesis 2>&1 > /dev/null
+
+./gcosmos genesis gentx val 1000000stake --chain-id=${CHAIN_ID} --home ./test/val1 --output-document ./test/genesis/config/gentx/1.json &> ./test/val1/gentx.log
+./gcosmos genesis gentx val 1000000stake --chain-id=${CHAIN_ID} --home ./test/val2 --output-document ./test/genesis/config/gentx/2.json &> ./test/val2/gentx.log
+./gcosmos genesis gentx val 1000000stake --chain-id=${CHAIN_ID} --home ./test/val3 --output-document ./test/genesis/config/gentx/3.json &> ./test/val3/gentx.log
+./gcosmos genesis gentx val 1000000stake --chain-id=${CHAIN_ID} --home ./test/val4 --output-document ./test/genesis/config/gentx/4.json &> ./test/val4/gentx.log
+
+# Collect all the gentxs to add the validators to the genesis file
+./gcosmos genesis collect-gentxs --home ./test/genesis 2>&1 > /dev/null
+
+# Distribute the final genesis.json to all the validators
+cp ./test/genesis/config/genesis.json ./test/val1/config/genesis.json 
+cp ./test/genesis/config/genesis.json ./test/val2/config/genesis.json
+cp ./test/genesis/config/genesis.json ./test/val3/config/genesis.json
+cp ./test/genesis/config/genesis.json ./test/val4/config/genesis.json
+
+# Start the seed node
+./gcosmos gordian seed ./test/p2p.seed.txt &
+
+# Wait for the seed node to start
+sleep 2
```

Comment:
We could add a command to block until the seed server is ready, so you don't have to wait extra time if it starts quickly. That is complex enough to warrant a separate PR.